exercises/ellipsoids.gie

```diff
@@ -1,18 +1,16 @@
-<gie>
-
 ###############################################################################
 
                        Specifying ellipsoid models
 
-In this exercise we will introduce how to set up the ellipsoid model used in
-projections and transformations. There are several ways to specify an ellipsoid
-model which we will familiarize ourselves with in this set of exercises.
+In this exercise we will introduce some of the many ways one can specify the
+ellipsoid model used in projections and transformations.
 
 We will be using the Mercator projection to illustrate the different ellipsoid
-settings. Note that ALL for all projections and some transformations the
-ellipsoid can be changed using the same parameter names.
+settings, but note thet the same parameter settings can be used for setting
```

Comment:
I believe that the ellipsoid is hardcoded in a few projections, hence the conservative wording

Reply:
Yes, case in point: bertin1953 and calcofi.

Reply:
Yes - calcofi is a bastard!